tests/common.go

```diff
@@ -5147,37 +5178,64 @@ func CreateMultiVolumesAndAttach(wg *sync.WaitGroup, count int, nodeName string)
 }
 
 //GetPoolIDWithIOs returns the pools with IOs happening
-func GetPoolIDWithIOs(pools map[string]*opsapi.StoragePool) (string, error) {
+func GetPoolIDWithIOs() (string, error) {
 	// pick a  pool doing some IOs from a pools list
-	for _, pool := range pools {
-		currentUsedSize := pool.Used
-		time.Sleep(3 * time.Second)
-		updatedPool, err := GetStoragePoolByUUID(pool.Uuid)
-		if err != nil {
+	var selectedPool *opsapi.StoragePool
+	var err error
+	stNodes := node.GetStorageNodes()
+	for _, stNode := range stNodes {
+		selectedPool, err = GetPoolWithIOsInGivenNode(stNode)
+		if err != nil && !strings.Contains(err.Error(), "no pools have IOs running") {
 			return "", err
 		}
-		newUsedSize := updatedPool.Used
-		if currentUsedSize != newUsedSize {
-			return pool.Uuid, nil
+		if selectedPool != nil {
+			return selectedPool.Uuid, nil
 		}
 	}
+
 	return "", fmt.Errorf("no pools have IOs running")
 }
 
+// GetPoolWithIOsInGivenNode returns the poolID in the given node with IOs happening
+func GetPoolWithIOsInGivenNode(stNode node.Node) (*opsapi.StoragePool, error) {
+
+	poolsDataBfr, err := Inst().V.GetPoolsUsedSize(&stNode)
+	if err != nil {
+		return nil, err
+	}
+
+	time.Sleep(5 * time.Second)
+
+	poolsDataAfr, err := Inst().V.GetPoolsUsedSize(&stNode)
+	if err != nil {
+		return nil, err
+	}
+
+	for k, v := range poolsDataBfr {
```

Comment:
can we have some easily readable names ?

Reply:
will update in next PR